docs/roles/integrator/integrating.md

```diff
@@ -0,0 +1,372 @@
+
+## Blockchain
+
+### Finality
+
+To learn whether a block is final or note, NEAR accepts a flag in calls to all RPC endpoints that indicates whether the requester wants the response as of the last block or the last *final* block.
+
+Exchanges should only use the last final block.
+
+<blockquote class="warning">
+<strong>work in progress</strong> <span>this feature is currently under development</span><br><br>
+
+Currently the list of parameters in the example below *does not include* the finality flag mentioned above.
+
+</blockquote>
+
+```sh
+# query format: account/<account_id>
+http post http://rpc.nearprotocol.com jsonrpc=2.0 method=query  \
+                                      params:="[\"account/test.near\",\"\"]" \
+                                      id="placeholder"
+# the request above returns view of account information
+# ------------------------------------------------------------------------------
+# response
+{
+  "id": "placeholder",
+  "jsonrpc": "2.0",
+  "result": {
+    "amount": "1000000000000000011",
+    "code_hash": "11111111111111111111111111111111",
+    "locked": "0",
+    "storage_paid_at": 0,
+    "storage_usage": 182
+  }
+}
+
+# this example used HTTPie - a CLI, cURL-like tool for humans available at http://httpie.org
+```
+
+#### Resources
+
+- RPC interface in related documentation [here](https://docs.nearprotocol.com/docs/interaction/rpc)
+- Consensus and finality here [NEAR Lunch and Learn Ep. 04: Nightshade: Consensus and finality](https://www.youtube.com/watch?v=k2ziZiZWquQ&list=PL9tzQn_TEuFW_t9QDzlQJZpEQnhcZte2y&index=4)
+
+### `/status` Endpoint
+
+The `/status` endpoint is available for querying the most current state of the blockchain.
+
+For example, TestNet status is available via `http://rpc.nearprotocol.com/status`
+
+An `HTTP GET` request to this endpoint produces the following output:
+
+```js
+{
+  "chain_id": "testnet", // universal name of the chain, configured in ~/.near/genesis.json
+  "rpc_addr": "0.0.0.0:8080",
+  "sync_info": {
+    "latest_block_hash": "9MZPsj3fzF2tQtPsZz12M1XKCWZHcZU1QFjBw9coHboW",
+    "latest_block_height": 12793,
+    "latest_block_time": "2019-12-17T07:01:09.474230160Z",
+    "latest_state_root": "FFrjV9UrH5gT4mwUYuat6xr19vVATWJstatnUj9V196z",
+    "syncing": false
+  },
+  "validators": [
+    {
+      "account_id": "far",
+      "is_slashed": false
+    }
+  ],
+  "version": {
+    "build": "c4046b02-modified",
+    "version": "0.4.10"
+  }
+}
+```
+
+## Accounts
+
+### Accounts
+
+The NEAR platform provides an account name system with a scoping structure similar to the domain name system.  You can read more about [accounts on the NEAR platform here](https://nomicon.io/Primitives/Account.html).
+
+Each account lives on one and only one shard.
+
+All related data for an account also lives on the same shard including:
+- Account Balance
+- Locked balance (for staking)
+- Code of the contract
+- Key-value storage of the contract. Stored in a ordered trie
```

Comment:
trie can't be ordered or unordered.
Also doesn't really matter in this case.

Reply:
will update.  note, this is copied directly from here: https://nomicon.io/Primitives/Account.html#account

Reply:
✅